readme-cmake.md

```diff
@@ -75,12 +75,13 @@ gcc-c++ cmake ninja-build SDL2-devel libpng-devel libjpeg-turbo-devel mesa-libGL
 Here's a summary of every command you will need for development:
 
 ```bash
-$ cmake --preset <preset>               # configure project (only needs to be done once)
-$ cmake --build --preset <preset>-debug # actually build Endless Sky (as well as any tests)
-$ ./build/<preset>/Debug/endless-sky    # run the game
-$ ctest --preset <preset>-test          # run the unit tests
-$ ctest --preset <preset>-benchmark     # run the benchmarks
-$ ctest --preset <preset>-integration   # run the integration tests (Linux only)
+$ cmake --preset <preset>                     # configure project (only needs to be done once)
+$ cmake --build --preset <preset>-debug       # actually build Endless Sky (as well as any tests)
+$ ./build/<preset>/Debug/endless-sky          # run the game
+$ ctest --preset <preset>-test                # run the unit tests
+$ ctest --preset <preset>-benchmark           # run the benchmarks
+$ ctest --preset <preset>-integration-debug   # run the integration tests in debug mode
```

Comment:
Why would anyone want to run _all_ integration tests in debug-mode?
I see a lot of value in running 1 or 2 integration tests in debug-mode, but running all of them in debug-mode really takes a long time, and I don' tsee why anyone would want to do that.

Reply:
Indeed running them all in debug mode doesn't really make sense. I'll update the docs to mention how to run individual integration tests instead.